client/src/pages/Login.js

```diff
@@ -15,6 +16,12 @@ const useStyles = makeStyles(theme => ({
     root: {
         alignItems: "center",
     },
+    box: {
+        marginTop: "3%",
```

Comment:
be careful with using percentages when calculating the positions in CSS because they may be inaccurate to what you expect it to be on different screen sizes